web/crux-ui/src/validation.ts

```diff
@@ -181,7 +179,7 @@ export const deploymentSchema = yup.object().shape({
   instances: yup.array(
     yup.object().shape({
       image: imageSchema,
-      config: explicitContainerConfigSchema.nullable(),
+      overridenConfig: explicitContainerConfigSchema.nullable(),
```

Comment:
There's a typo

Reply:
@m8vago is this type? I can rename if needed.

Reply:
`overridden` has two 'd'-s 🤷

Reply:
yeah rename it please